javaparser-core/src/main/javacc-support/com/github/javaparser/AutoCloseableProvider.java

```diff
@@ -0,0 +1,4 @@
+package com.github.javaparser;
+
+public interface AutoCloseableProvider extends Provider, AutoCloseable {
```

Comment:
I know why you're doing it this way, but damn, now we have "AutoCloseable" all over the API :-/

Reply:
Yep, tell me how to modify the JavaCC generated file instead. :-)
With Gradle it would be easy as pie to just post-process the generated file, e. g. with JavaParser.